matter_server/client/client.py

```diff
@@ -332,6 +332,11 @@ async def node_diagnostics(self, node_id: int) -> NodeDiagnostics:
                 continue
             mac_address = convert_mac_address(network_interface.hardwareAddress)
             break
+        else:
+            self.logger.warning(
+                "Could not determine network_interface info for Node %s, "
+                "is it missing the GeneralDiagnostics/NetworkInterfaces Attribute?"
+            )
```

Comment:
We're missing a log parameter for the node id.

Reply:
fixed